ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml

```diff
@@ -136,6 +136,59 @@ presubmits:
           - configMap:
               name: cluster-profile-gcp
     trigger: ((?m)^/test( all| gcp),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - release-3.10
+    - dockerless-build-temporary
+    context: ci/prow/atomic
+    decorate: true
+    name: pull-ci-openshift-openshift-ansible-release-3.10-e2e-atomic
+    rerun_command: /test atomic
+    skip_cloning: true
+    spec:
+      containers:
+      - command:
```

Comment:
Please have `ci-operator` as your only `command` and the rest in `args`

Reply:
Done